learn/dapps-list/dapp-directory.md

```diff
@@ -0,0 +1,299 @@
+---
+title: List your Dapp on the Moonbeam DApp Directory
+description: Follow this tutorial to learn how to list your Moonbeam or Moonriver project or update a current listing on the Moonbeam Foundation DApp Directory.
+---
+
+# How to List your Project on the Moonbeam DApp Directory
+
+## Introduction to The Moonbeam DApp Directory {: #introduction-to-state-of-the-dapps }
+
+The Moonbeam ecosystem comprises two distinct networks: Moonbeam and Moonriver. Each network has its own dedicated [DApp Directory](https://apps.moonbeam.network/moonbeam/app-dir){target=\_blank}, maintained by the Moonbeam Foundation. These directories categorize projects spanning from DeFi to NFTs to gaming, providing users with comprehensive access to diverse applications.
+
+You'll supply core project details like name, description, and relevant links when adding your project. Depending on your project type, you may include additional data such as on-chain stats and token information.
+
+Despite the distinction between the Moonbeam and Moonriver DApp directories, the submission process remains the same. To list your project on the DApp Directory, you must submit a pull request to the [Moonbeam Foundation's App Directory Data repository on GitHub](https://github.com/moonbeam-foundation/app-directory-data){target=\_blank}. This guide outlines the necessary data and formatting specifics for your submission.
```

Comment:
This Github repo is currently private, not sure if that needs to be changed before we go live with the docs?

Reply:
Yeah, it'll be coordinated so that when we release the docs it'll be public